collections/_article/dev-snapshot-godot-4-2-dev-3.md

```diff
@@ -0,0 +1,149 @@
+---
+title: "Dev snapshot: Godot 4.2 dev 3"
+excerpt: "One month into the release cycle for Godot 4.2, things are well under way. This third dev snapshot features a lot of improvements to C# and rendering."
+categories: ["pre-release"]
+author: Rémi Verschelde
+image: /storage/blog/covers/dev-snapshot-godot-4-2-dev-3.webp
+image_caption_title: ""
+image_caption_description: ""
+date: 2023-08-11 13:00:00
+---
+
+Development is well under way for our next milestone, Godot 4.2. After our [dev 2 snapshot](/article/dev-snapshot-godot-4-2-dev-2/), we kept merging a number of new features and important bug fixes, so we're ready for a third dev snapshot to put those under broader user testing. It's been around a month since we started this release cycle, with roughly two months to go for 4.2 beta 1, and a fourth month to reach stable in early November 2023.
+
+I'd like to outline the amazing work done by the [Production team](https://godotengine.org/teams/#production), and especially by Yuri Sizov, who took care of PR reviews, merges, and the whole release process while I was on holiday in the second half of July. Godot 4.1.1, the first two dev snapshots for 4.2, and 4.0.4, were all fully handled by Yuri, while I used to be the sole responsible for making and publishing releases. We're now fully sharing the role of release manager, and that should enable us to keep this sustained pace with a high quality standard.
```

Comment:
❤️

```suggestion
I'd like to outline the amazing work done by the [Production team](https://godotengine.org/teams/#production), and especially by Yuri Sizov, who took care of PR reviews, merges, and the whole release process while I was on holiday in the second half of July. Godot 4.1.1, the first two dev snapshots for 4.2, and 4.0.4, — were all fully handled by Yuri. While I used to be the sole responsible for making and publishing releases, we're now fully sharing the role of release manager. That should enable us to keep this sustained pace with a high quality standard.
```

I'm not sure about the "while I used to be" and how it fits in the previous sentence, so I suggest separating it.